### ISSUE TITLE
`helm plugin install` failing on git v2.16.0+

### ISSUE THREAD

**Issue by bacongobbler** [kubernetes/helm#3509]
As of git v2.16.0, [git returns an error](https://github.com/git/git/blob/master/Documentation/RelNotes/2.16.0.txt#L15-L18) when you perform a `git checkout` with no specified branch name, as it assumes the user meant to check out "everything".

In helm, this manifests in the following error:

```
><> helm plugin install https://github.com/bacongobbler/helm-whatup
Error: Unable to update checked out version
```

The workaround is to provide a string value to Masterminds/vcs, which can be done via

```
><> helm plugin install https://github.com/bacongobbler/helm-whatup --version v0.1.0
```

I have submitted a patch upstream via https://github.com/Masterminds/vcs/pull/86, but I'm posting it here for visibility on how this error can pop up.

The resolution is to either stay on git v2.15 or to use the demonstrated workaround shown above.

cc @mattfarina

**Comment by bacongobbler** [comment]
also related: Azure/draft#522

**Comment by mattfarina** [comment]
@bacongobbler is the goal of this to checkout the tip of master when no version is specified or the latest release?

**Comment by bacongobbler** [comment]
Closed via #3571

The former :)
